templates/klage-dittnav-pdfgen/klage.hbs

```diff
@@ -161,6 +161,14 @@
                     <p>{{adresse}}</p>
                 </div>
             </div>
+            {{#if fullmektigNavn}}
+            <div class="single-row">
+                <div class="info-desc">Fullmakt</div>
+                <div class="info-data">
+                    <p>Klagen er innsendt av fullmektig: {{fullmektigNavn}}, fødselsnummer: {{fullmektigFnr}}</p>
```

Comment:
Burde vi sjekke om `fullmektigFnr` finnes også?

Reply:
Kan godt gjøre det for lokal fullstendighet, men vi sikrer at begge fins i kallende tjenester.

Reply:
Ser ikke ut til at `AND` er støttet av handlebars uten videre, så da tenker jeg at det kan droppes: https://stackoverflow.com/questions/8853396/logical-operator-in-a-handlebars-js-if-conditional

Reply:
Stemmer, enkleste løsning er da å bruke `{{#if }}` i to nivåer, men det er ikke veldig pent.
Et annet alternativ som er relativt vanlig med Handlebars er å beregne slikt på forhånd i en ekstra egenskap;
```
{
  hasFullmektig: fullmektigFnr && fullmektigNavn
}
```
Men det er kanskje ikke verdt det?

Reply:
Jeg tenker at det ikke er nødvendig å legge inn en sånn sikring. Pdfgen krasjer ikke om den mangler, og slik vi bruker disse feltene så opptrer de alltid sammen.